vendor/jquery.dataTables-1.9.4.js

```diff
@@ -6487,6 +6487,9 @@
 			_fnMap( oSettings, oInit, "oSearch", "oPreviousSearch" );
 			_fnMap( oSettings, oInit, "aoSearchCols", "aoPreSearchCols" );
 			_fnMap( oSettings, oInit, "iDisplayLength", "_iDisplayLength" );
+      _fnMap( oSettings, oInit, "iDisplayStart", "_iDisplayStart" );
+      _fnMap( oSettings, oInit, "iRecordsTotal", "_iRecordsTotal" );
+      _fnMap( oSettings, oInit, "iRecordsDisplay", "_iRecordsDisplay" );
```

Comment:
nit: spacing is off here

Reply:
Weird, it's not showing up in RubyMine as having weird spacing.